app/mobile/lib/presentation/pages/forum_page.dart

```diff
@@ -15,31 +17,43 @@ class ForumPage extends StatefulWidget {
 }
 
 class _ForumPageState extends State<ForumPage> {
-  var isLoggedIn = true;
+  late bool isLoggedIn;
+
+  @override
+  void initState() {
+    super.initState();
+    try {
+      isLoggedIn = true;
+      CacheManager().getUser();
+    } catch (e) {
+      isLoggedIn = false;
+    }
+  }
```

Comment:
I don't think this is a good way of checking if user is logged in. We can do something like this: 
```
bool isLoggedIn = SharedManager().checkString(SharedKeys.sessionId); 
```
It would be a lot better if you convert all the declarations to this usage.

Reply:
I fixed and tested.